subprojects/docs/src/docs/userguide/dependencyManagementForJavaProjects.adoc

```diff
@@ -0,0 +1,109 @@
+// Copyright 2017 the original author or authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+[[dependency_management_for_java_projects]]
+== Dependency Management for Java Projects
+
+This chapter explains how to apply basic dependency management concepts to Java-based projects. For a detailed introduction to dependency management, see <<introduction_dependency_management>>.
+
+[[sec:setting_up_a_standard_build_script_java_tutorial]]
+=== Dissecting a typical build script
+
+Let's have a look at a very simple build script for a Java-based project. It applies the <<java_library_plugin,Java Library plugin>> which automatically introduces a standard project layout, provides tasks for performing typical work and adequate support for dependency management.
+
+++++
+<sample id="basicDependencyDeclarations" dir="userguide/artifacts/dependencyBasics" title="Dependency declarations for Java-based project">
+    <sourcefile file="build.gradle"/>
+</sample>
+++++
+
+The api:org.gradle.api.Project#dependencies[] code block declares that Hibernate core 3.6.7.Final is required to compile the project's production source code. It also states that junit &gt;= 4.0 is required to compile the project's tests. All dependencies are supposed to be looked up in the Maven Central repository as defined by api:org.gradle.api.Project#repositories[]. The following sections explain each aspect in more detail.
+
+[[sec:module_dependencies_java_tutorial]]
+=== Declaring module dependencies
+
+There are various <<dependency_types,types of dependencies>> that you can declare. One such type is an _module dependency_. A <<sec:declaring_dependency_to_module,module dependency>> represents a dependency on a module with a specific version built outside the current build. Modules are usually stored in a repository, such as Maven Central, a corporate Maven or Ivy repository, or a directory in the local file system.
```

Comment:
"One such type is an module..." -> "One such type is a module..."

Reply:
done